examples/thread-duell/README.md

```diff
@@ -0,0 +1,9 @@
+Thread_Duel
```

Comment:
the example name should reflect single-"l" "duel". Also, we usually use "_" vs "-".

Reply:
renamed